worker/orca_grader/__main__.py

```diff
@@ -219,15 +218,16 @@ def handle_grading_job(grading_job: GradingJobJSON, image_name: str | None = Non
         else:
             warn_str = "The processes timed out during execution." if \
                         result.did_timeout else "An error was encountered during execution."
-            _LOGGER.warn(warn_str)
+            _LOGGER.warning(warn_str)
         _LOGGER.debug("\n".join(result.results))
 
 
 def inform_client_of_reenqueue(grading_job: GradingJobJSON,
                                updated_db_id: int) -> None:
     post_job_status_to_client(location="Queue",
                               response_url=grading_job["response_url"],
-                              key=grading_job["key"])
+                              key=grading_job["key"],
+                              id=updated_db_id)
```

Comment:
This seems unrelated to this PR's purpose -- was this just a bug?

Reply:
Yup -- this needed to be passed in  to communicate the ID appropriately to bottlenose so it knew how to get the proper status

Reply:
I've made a note of this in the PR description in case we need to revisit it later